src/GitHub.Api/Platform/DefaultEnvironment.cs

```diff
@@ -141,8 +143,8 @@ public NPath NodeJsExecutablePath
                 if (!nodeJsExecutablePath.IsInitialized)
                 {
                     nodeJsExecutablePath = IsWindows
-                        ? UnityApplication.Parent.Combine("Data", "Tools", "nodejs", "node.exe")
-                        : UnityApplication.Combine("Contents", "Tools", "nodejs", "node");
+                        ? UnityApplicationContents.Parent.Combine("Tools", "nodejs", "node.exe")
```

Comment:
This isn't right, `EditorApplication.applicationContentsPath` points to the `Editor\Data` directory which contains the `Tools` directory. It's the same logic as on the mac, in fact.

Reply:
I dunno how that `Parent` got there...